src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -94,6 +94,7 @@
     private AuthenticationUpdater authenticationUpdater = null;
     private final Set<ConnectionListener> listeners = new CopyOnWriteArraySet<ConnectionListener>();
     private int sshRxBufferSize = SSH_RX_BUFFER_SIZE;
+    private StringBuilder eventBuffer = null;
```

Comment:
Please run this code then check displayed object hash. It is generated within private method then stored into instance field.
https://gist.github.com/rinrinne/7e40638fba102d52b1d34a7b70b16b2d